src/main/scala/me/rexim/issuestant/SplitIssueTree.scala

```diff
@@ -0,0 +1,13 @@
+package me.rexim.issuestant
+
+object SplitIssueTree {
+  def fromIssue(issue: Issue) = SplitIssueTree(issue.number, issue.title, issue.html_url, Nil)
+}
+
+case class SplitIssueTree(number: Int, title: String, htmlUrl: String, children: List[SplitIssueTree]) {
+  def print(level: Int = 0): Unit = {
```

Comment:
No need for that. This function is for debug output only. Plan to get rid of it in the future.